itou/templates/signup/signup_job_seeker.html

```diff
@@ -36,6 +36,8 @@ <h1>
         {% bootstrap_field form.city %}
     </div>
 
+    {% bootstrap_field form.resume_link %}
```

Comment:
À titre personnel, je dégagerais tous les champs optionnels du formulaire d'inscription afin d'aérer et de pouvoir aller droit au but pendant cette phase.

Reply:
Tu veux dire adresse, CV ? Et ne pouvoir y accéder et les modifier que via le dashboard ?

Reply:
Oui c'est ça ! En plus ils sont demandés dans le process de candidature si non renseignés il me semble.

Reply:
Ok je vais vérifier. On fait une passe avec le métier sur le sujet ?

Reply:
Up to you. Moi j'ai tendance à responsabiliser les devs en général

Reply:
Pas de passe métier, on est des vikings

Quelque chose comme ça, donc:

![simple-job-seeker-form](https://user-images.githubusercontent.com/147232/80715094-283e9780-8af6-11ea-942a-91141e76395e.png)